aisynphys/ui/notebook.py

```diff
@@ -148,13 +150,17 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
         List of postsynaptic cell classes in the order they should be displayed
     class_labels : dict
         Maps {cell_class: label} to give the strings to display for each cell class.
-    distance_adjusted : bool
-        If True, use distance-adjusted connectivity metrics. See 
-        ``aisynphys.connectivity.measure_connectivity(sigma)``.
     cmap : matplotlib colormap instance
         The colormap used to generate colors for each matrix element
     norm : matplotlib normalize instance
         Normalize instance used to normalize connection probability values before color mapping
+    ctype: string
+        corresponds to dict key from ``aisynphys.connectivity.measure_connectivity``
+        distance_adjusted - use distance-adjusted connectivity metrics.
```

Comment:
ctype and distance_adjusted are orthogonal--how would you do distance-adjusted gap junction connectivity?

Reply:
I guess I assumed the the distance-adjusted was not going to be broadly applicable like that, but we could split out into ctype='chemical' or 'electrical', and then have distance-adjusted go back to being a bool